### ISSUE TITLE
[Bug] TokenAcquisition.GetAuthenticationResultForUserAsync should allow TokenAcquisitionOptions for calls from web app

### ISSUE THREAD

**Issue by dandax** [AzureAD/microsoft-identity-web#1634]
**Is your feature request related to a problem? Please describe.**
The `TokenAcquisition.GetAuthenticationResultForUserAsync` accepts `TokenAcquisitionOptions` as a parameter but will only use them when a call is coming from a web API. The code specifically passes `null` to the `tokenAcqusitionOptions` parameter:

``` csharp
// If access token is null, this is a web app
authenticationResult = await GetAuthenticationResultForWebAppWithAccountFromCacheAsync(
     application,
     user,
     scopes,
     tenantId,
     mergedOptions,
     userFlow,
     null) // <-- should pass tokenAcqusitionOptions here
     .ConfigureAwait(false);
```

**Describe the solution you'd like**
I would like to see the parameter used for web app calls as well.

**Describe alternatives you've considered**
The only alternative I can see right now is to create my own `IConfidentialClientApplication` and apply my `TokenAcqusitionOptions` to the builder - effectively a copy of `GetAuthenticationResultForWebAppWithAccountFromCacheAsync`.

**Additional context**
Our app allows a user to update their first and last name in their B2C profile. Specifically, I'm trying to use `ForceRefresh` to get a new token after we make the Graph API call.

**Comment by jmprieur** [comment]
Thanks for the heads up @dandax

**Comment by dandax** [comment]
Wanted to see if there's anyway this can be bumped up. The project it was assigned to hasn't been updated since March. Seems like an easy enough fix to get in an upcoming release.

**Comment by jmprieur** [comment]
@dandax: this was fixed in February (in 1.23.1). 
Do you still see the issue in the latest Id.Web?
